dashboard/app/models/sections/section.rb

```diff
@@ -429,7 +434,7 @@ def reset_code_review_groups(new_groups)
   end
 
   def update_code_review_expiration(enable_code_review)
-    self.code_review_expires_at = enable_code_review ? DateTime.now + 90.days : nil
+    self.code_review_expires_at = enable_code_review ? Time.now.utc + 90.days : nil
```

Comment:
how does `Time.now` differ from `DateTime.now`? I may need to update the ux to parse the date differently

Reply:
I don't think it matters -- mostly wanted to do the utc thing explicitly because in development envs where system time is something other than UTC, you can get some weird timestamp things happening if you aren't explicit. To confirm, I stored `Time.now.utc + 90.days` as a section's `code_review_expires_at` and copied into JS console (ignore the console error :)):

![image](https://user-images.githubusercontent.com/25372625/142667115-a5ef2901-092f-404d-9d7f-454eae5bccd1.png)